pkg/chartutil/create.go

```diff
@@ -231,7 +240,9 @@ metadata:
   labels:
     {{- include "<CHARTNAME>.labels" . | nindent 4 }}
 spec:
+{{- if not .Values.autoscaling.enabled }}
```

Comment:
PS this is an especially important change that got rolled out to the stable charts repo. Noted in #7369

Reply:
👍